images/alpine/Dockerfile

```diff
@@ -12,7 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM alpine:latest
+# Update the date below (check https://hub.docker.com/_/alpine/tags) to get the
+# alpine edge base image, which has a faster release cadence than alpine:latest
+# (which tracks alpine stable).
```

Comment:
Maybe note the reason of getting security updates sooner?

Reply:
"Secure software is better than unsecure software"?

But I don't think that's what you meant, so, maybe you could add a suggestion directly. See step 7 in https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request.

Reply:
```suggestion
# (which tracks alpine stable), to pick up security patches more quickly.
```

And sorry, didn't mean the reason "for" getting security updates sooner; meant "of", to anticipate the question, "Why don't we prefer stable, isn't edge dangerous?"

Reply:
Ah, makes sense. Will update in all places.

Reply:
Done.